screenutils/screen.py

```diff
@@ -99,6 +100,15 @@ def exists(self):
         return self.name in [".".join(l.split(".")[1:]).split("\t")[0]
                              for l in lines]
 
+    @property
+    def pid(self):
+        lines = getoutput("screen -ls | grep " + self.name).split('\n')
```

Comment:
instead of a grep, I would rather add in the for loop `if self.name not in l: continue`